src/Lunr/Ticks/PHPMailer/Tests/PHPMailerTestCase.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * This file contains the PHPMailerTest class.
+ *
+ * SPDX-FileCopyrightText: Copyright 2025 Framna Netherlands B.V., Zwolle, The Netherlands
+ * SPDX-License-Identifier: MIT
+ */
+
+namespace Lunr\Ticks\PHPMailer\Tests;
+
+use Lunr\Halo\LunrBaseTestCase;
+use Lunr\Ticks\EventLogging\EventInterface;
+use Lunr\Ticks\EventLogging\EventLoggerInterface;
+use Lunr\Ticks\PHPMailer\PHPMailer;
+use Lunr\Ticks\TracingControllerInterface;
+use Lunr\Ticks\TracingInfoInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+
+/**
+ * This class contains common setup routines, providers
+ * and shared attributes for testing the PHPMailerTest class.
+ *
+ * @covers Lunr\Ticks\PHPMailer\PHPMailer
+ */
+abstract class PHPMailerTestCase extends LunrBaseTestCase
+{
+
+    /**
+     * Instance of the EventLoggerInterface.
+     * @var EventLoggerInterface&MockObject
+     */
+    protected EventLoggerInterface&MockObject $logger;
+
+    /**
+     * Mock instance of a Tracing Controller
+     * @var TracingControllerInterface&TracingInfoInterface&MockObject
+     */
+    protected TracingControllerInterface&TracingInfoInterface&MockObject $controller;
+
+    /**
+     * Instance of the EventInterface.
+     * @var EventInterface&MockObject
+     */
+    protected EventInterface&MockObject $event;
+
+    /**
+     * Instance of the tested class.
+     * @var PHPMailer
+     */
+    protected PHPMailer $class;
+
+    /**
+     * TestCase Constructor.
+     */
+    public function setUp(): void
+    {
+        $this->event = $this->getMockBuilder(EventInterface::class)->getMock();
+
+        $this->logger = $this->getMockBuilder(EventLoggerInterface::class)->getMock();
+
+        $this->controller = $this->createMockForIntersectionOfInterfaces(
+            [
+                TracingControllerInterface::class,
+                TracingInfoInterface::class,
+            ]
+        );
```

Comment:
You can try. It very much depends on how you're going to use it later. If there's any case where you'd need `withConsecutive` or some such you'd have to switch to Mockery